src/_pytest/_code/code.py

```diff
@@ -56,6 +56,12 @@
 
     _TracebackStyle = Literal["long", "short", "line", "no", "native", "value", "auto"]
 
+BaseExceptionGroup: Type[BaseException]
```

Comment:
I suggest we use an explicit version check here:

```python
if sys.version_info[:2] < (3, 11):
    from exceptiongroup import BaseExceptionGroup
```

This makes it easy to remove this code once it is no longer needed, also it won't given a confusing error message in case we are running in corrupted environment.

Reply:
yeah that's much cleaner